shared/fs/footer/upload-container.tsx

```diff
@@ -68,7 +68,7 @@ const mergeProps = ({_kbfsDaemonStatus, _pathItems, _uploads}, {debugToggleShow}
     ...uploadsToUploadCountdownHOCProps(_pathItems, _uploads),
     debugToggleShow,
     isOnline:
-      !flags.kbfsOfflineMode || _kbfsDaemonStatus.onlineStatus === Types.KbfsDaemonOnlineStatus.Online,
```

Comment:
is there a ticket to clean up the offline mode flag? if not can you make one?

Reply:
I'll just do it in this PR. Thanks!